src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -20,10 +21,10 @@
 
 namespace Akka.Remote.Tests
 {
-    
+
     public class RemotingSpec : AkkaSpec
     {
-        public RemotingSpec():base(GetConfig())
+        public RemotingSpec() : base(GetConfig())
```

Comment:
Your comments about "why you have not implemented the specific spec" could help to future generations